SonarTfsAnnotate/Program.cs

```diff
@@ -73,16 +73,20 @@ static int Main(string[] args)
                         continue;
                     }
 
+                    WorkspaceInfo workspaceInfo = Workstation.Current.GetLocalWorkspaceInfo(path);
```

Comment:
@demuruge It seems the user on StackOverflow is indeed facing a NPE with this PR. Do you have an idea of a proper fix?